pipelines/tekton/aiedge-e2e/aiedge-e2e.pipeline.yaml

```diff
@@ -208,7 +208,7 @@ spec:
     - name: IMAGE
       value: image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/$(params.model-name):$(params.model-version)
     - name: BUILDER_IMAGE
-      value: registry.redhat.io/rhel8/buildah@sha256:0a86ecbdfbe86e9d225b7fe4b090a0dd6d323f8afdfdf2bd933ff223ddb53320
+      value: registry.redhat.io/rhel8/buildah@sha256:f5521de9445a04054c386707b774d7d534556c03b29ba4c91cfddfa979cf761c
```

Comment:
/me wonders if we should be using UBI rather than RHEL, and 9 instead of 8 (not that it would fix this actual issue)

Reply:
I agree on the switch to ubi9 for parity with upstream implementation

Without regular automation to update the digest hash, we should switch to the tag.

We can follow up in other PRs to address the changes

Reply:
OK, I will create a jira for that.

Reply:
Created [RHOAIENG-3359](https://issues.redhat.com/browse/RHOAIENG-3359)